src/ema_cross_strategy.py

```diff
@@ -18,10 +19,17 @@ def ema_cross_strategy(symbol, timeframe, short_term_ema_length, long_term_ema_l
     # Append indicator columns to dataframe
     calculate_indicators(ema_x_strategy_table, short_term_ema_length, long_term_ema_length)
 
-    # Determine if a trade event has occurred
-    det_trade(ema_x_strategy_table, short_term_ema_length, long_term_ema_length)
+    trade_event = ema_x_strategy_table.tail(1).copy()
 
-    return ema_x_strategy_table
+    if trade_event['ema_cross'].values:
+        new_trade = True
+        comment = f"EMA_Cross_strategy_{symbol}"
+        print(comment)
```

Comment:
Do we want this print long-term or was this here for short-term testing?

Reply:
Short-term. I'll remove it.